coops-ui/src/views/AnnualReport.vue

```diff
@@ -452,17 +453,17 @@ export default {
             }
 
             // load Change of Address fields
-            const changeOfAddress = filing.changeOfAddress
-            if (changeOfAddress) {
-              if (changeOfAddress.deliveryAddress && changeOfAddress.mailingAddress) {
-                this.addresses = {
-                  deliveryAddress: changeOfAddress.deliveryAddress,
-                  mailingAddress: changeOfAddress.mailingAddress
+            const changeOfAddress = filing.changeOfAddress.offices
```

Comment:
1. I think you need a check that filing.changeOfAddress is not null before you dereference office.
2. You should change the const name to offices.
3. Isn't code needed below to handle recordsOffice?

Reply:
1. Could certainly add this check!
2. Sure can do, was just following the pattern defined earlier when directors are defined
line 425 `const changeOfDirectors = filing.changeOfDirectors`. Would you prefer that to be refactored as well to `directors`?
3. Bcorps cannot file office address changes through the annual report flow.

Reply:
1. Could certainly add this check!
2. Sure can do, was just following the pattern defined earlier when directors are defined
line 425 `const changeOfDirectors = filing.changeOfDirectors`. Would you prefer that to be refactored as well to `directors`?
3. Bcorps cannot file office address changes through the annual report flow.

Reply:
Re: 2.

The pattern is that the const is a shortcut to the property, so `const changeOfDirectors = filing.changeOfDirectors` makes sense. For this reason, on line 456 above, I think it should be `const offices = filing.changeOfAddress.offices`.

Reply:
Cool, done and done 👍